cpp/src/neighbors/detail/nn_descent.cuh

```diff
@@ -491,11 +491,11 @@ __device__ __forceinline__ void remove_duplicates(
 template <typename Index_t, typename ID_t = InternalID_t<Index_t>>
 RAFT_KERNEL
 #ifdef __CUDA_ARCH__
-#if (__CUDA_ARCH__) == 750 || ((__CUDA_ARCH__) >= 860 && (__CUDA_ARCH__) <= 890) || \
-  (__CUDA_ARCH__) == 1200
-__launch_bounds__(BLOCK_SIZE)
-#else
+#if (__CUDA_ARCH__) == 700 || (__CUDA_ARCH__) == \
+  800 || (__CUDA_ARCH__) == 900 || (__CUDA_ARCH__) == 1000
```

Comment:
Style checks now complain after I fixed my typo. Correcting that:
```suggestion
#if (__CUDA_ARCH__) == 700 || (__CUDA_ARCH__) == 800 || (__CUDA_ARCH__) == 900 || \
  (__CUDA_ARCH__) == 1000
```

Reply:
Sorry I missed that too!